.travis.yml

```diff
@@ -0,0 +1,40 @@
+sudo: false
+language: scala
+scala:
+  - 2.10.6
+  - 2.11.11
+  - 2.12.2
+jdk:
+  - oraclejdk8
+install:
+  - mkdir -p $HOME/.sbt/0.13/plugins/
+  - echo 'addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.0-RC3")' > $HOME/.sbt/0.13/plugins/coursier.sbt
+  - if [[ "${TRAVIS_SCALA_VERSION}" == "2.10.6" ]]; then TEST_SBT_PLUGIN=true; else TEST_SBT_PLUGIN=false; fi
+  - git clone https://github.com/scala-js/scala-js.git
+  - cd scala-js
+  - git checkout f21e7f5ea652fe4a186f1ecd8c0070fbb80edc80
+  - sbt ++$TRAVIS_SCALA_VERSION ir/publishLocal tools/publishLocal jsEnvs/publishLocal jsEnvsTestKit/publishLocal
+  - |
+      if [[ "${TEST_SBT_PLUGIN}" == "true" ]]; then
+        sbt ++$TRAVIS_SCALA_VERSION testAdapter/publishLocal sbtPlugin/publishLocal &&
+        sbt ++2.11.11 compiler/publishLocal library/publishLocal testInterface/publishLocal
+      fi
+  - cd ..
+script:
+  - sbt ++$TRAVIS_SCALA_VERSION scalajs-phantomjs-env/test scalajs-phantomjs-env/doc
+  - |
+      if [[ "${TEST_SBT_PLUGIN}" == "true" ]]; then
+        sbt scalajs-phantomjs-env/publishLocal sbt-scalajs-env-phantomjs/publishLocal && \
+        cd sbt-plugin-test && \
+        sbt jetty9/run && \
+        cd ..
+      fi
+cache:
+  directories:
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt
+    - $HOME/.coursier/cache
```

Comment:
No they don't ;) (I checked). Even if they did, each new `publishLocal` would override the previous thing anyway, and they would have to use the updated one, because otherwise it would simply be unbearable even for local iterative development.